### ISSUE TITLE
Problem Pickling when using the createtimestamp or modifyTimestamp attribute

### ISSUE THREAD

**Issue by dangelsaurus** [cannatag/ldap3#376]
```
from ldap3 import Server, Connection, ALL
import pickle

server = Server('ipa.demo1.freeipa.org', get_info=ALL)
conn = Connection(server, 'uid=admin,cn=users,cn=accounts,dc=demo1,dc=freeipa,dc=org', 'Secret123', auto_bind=True)
entries = conn.extend.standard.paged_search('dc=demo1,dc=freeipa,dc=org', '(objectClass=person)', attributes=['cn', 'givenName', 'createtimestamp'], paged_size=5)

testlist = []

for entry in entries:
    testlist.append(entry)

output = open('data.pkl', 'wb')
pickle.dump(testlist, output)
output.close()

pkl_file = open('data.pkl', 'rb')
test2 = pickle.load(pkl_file)

pkl_file.close()
```

gives...

```
C:\Python36\python.exe C:/Users/xxxxx/Dropbox/xxx/Work/Python/CEDAR/ldaptest.py
Traceback (most recent call last):
  File "C:/Users/xxxxx/Dropbox/xxx/Work/Python/CEDAR/ldaptest.py", line 20, in <module>
    test2 = pickle.load(pkl_file)
TypeError: __init__() missing 2 required positional arguments: 'offset' and 'name'

Process finished with exit code 1
```

this looks to be related to ldap3.core.timezone.py... unfortunately this is where I don't have a clue what to do next...  Is this something that should be addressed before pickling with the datetime object itself?

**Comment by dangelsaurus** [comment]
related? https://stackoverflow.com/questions/36238351/python-pickle-loads-failed-for-class-instance

**Comment by cannatag** [comment]
Probably. Will work on this in the weekend.

Thanks,
Giovanni

**Comment by cannatag** [comment]
Please check code in dev. It should works now.

**Comment by cannatag** [comment]
Fixed in release 2.3 of ldap3.
